website/docs/roadmap.md

```diff
@@ -2,8 +2,20 @@
 
 We continuously work on updating our roadmap and we love to discuss those with our community. Feel encouraged to participate.
 
+## Q1 2024
+
+* 🎊 Supports embedding api for document context related use cases: https://github.com/TabbyML/tabby/issues/790
+* 🔐 SSO / OAuth support in Tabby EE: https://github.com/TabbyML/tabby/issues/1039
```

Comment:
How about `authorization` for EE? Like owner/admin/member access permissions. Feasible for Q1?

Reply:
should be feasible - so far we only have two tiers of authorization (admin / non-admin)